hls4ml/model/profiling.py

```diff
@@ -589,6 +589,7 @@ def get_ymodel_keras(keras_model, X):
         name = layer.name
         if (
             hasattr(layer, "activation")
+            and layer.activation is not None
             and layer.activation.__name__ != "linear"
```

Comment:
I think this line needs to move below the check of what type the layer is (i.e. swap 593 and 594)